local/gugcat/renderer.php

```diff
@@ -139,21 +138,41 @@ public function display_add_grade_form($course, $student, $gradeversions) {
         foreach($gradeversions as $gradeversion){
             $html .= html_writer::start_tag('div', array('class'=>'form-group row'));
             $html .= html_writer::start_tag('div', array('class'=> 'col-md-3'));
-            if ($gradeversion->itemname == get_string('moodlegrade', 'local_gugcat'))
-                $html .= html_writer::tag('label', $gradeversion->itemname. date(" [j/n/Y]", strtotime(userdate($gradeversion->timemodified))));
-            else 
-                $html .= html_writer::tag('label', $gradeversion->itemname);
+            $html .= html_writer::tag('label', $gradeversion->itemname);
             $html .= html_writer::end_tag('div');
-            $html .= html_writer::div(local_gugcat::convert_grade($gradeversion->grades[$student->id]->grade), 'col-md-9 form-inline felement');
```

Comment:
Latest codes were deleted.

Reply:
should be '**->grade**' instead of '->finalgrade'

Reply:
![image](https://user-images.githubusercontent.com/74167293/104007993-c20d2700-51e3-11eb-9929-603368345abd.png)
**Still, not resolve. This should be '->grade' instead of '->finalgrade'**